yo/blockchain_follower.py

```diff
@@ -48,7 +48,8 @@ def __init__(self,**kwargs):
     async def store_notification(self, **data):
         data['sent'] = False
         self.db.create_notification(**data)
-        
```

Comment:
The only concern of the blockchain follower is storing the notifications in the database. It should be unaware and unconcerned with whatever subsequent events may be triggered by those notifications.

Reply:
How does the notification sender send those notifications if it's not told to?
We could have it poll the database, but that's a really bad idea, hence the design of having the blockchain follower tell the notification sender.

Do you have a suggestion for a better design approach here?